airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -417,6 +423,8 @@ def create_datetime_based_cursor(self, model: DatetimeBasedCursorModel, config:
             model.start_datetime if isinstance(model.start_datetime, str) else self.create_min_max_datetime(model.start_datetime, config)
         )
         end_datetime = None
+        if model.is_data_feed and model.end_datetime:
+            raise ValueError("Data feed does not support end_datetime")
```

Comment:
@flash1293 This will probably require some work on the front end as I think the frontend is populating a default value

Reply:
See https://github.com/airbytehq/airbyte-platform-internal/pull/7485